.ci/docker/theo/src/commands/edgar.rs

```diff
@@ -28,12 +30,14 @@ impl TestEdgarCli {
     pub(crate) fn default_handling(&self) -> crate::Result {
         match self.task {
             TaskCli::Start => {
+                load_edgar_kernel_modules()?;
```

Comment:
This assumes THEO is running in privileged mode. This is not the case in the virtual machine nor the THEO setup with docker. For the virtual machine and testing purposes, we can configure it to always load the modules at startup. For the THEO docker setup on the host I'd rather have it use the sudo crate and not run as root all the time.

Reply:
THEO now checks if all kernel modules needed by EDGAR are loaded and, if not, loads missing kernel modules by invoking `sudo modprobe` as a subprocess. As `stdin` is inherited from THEO, the user can simply type in their password in this case.